doc/making-a-visualizer.md

```diff
@@ -86,12 +84,13 @@ won't be able to prompt the user for corrections at that point.
 -   **Differences:** Make sure that the number of terms is at least the number
     of levels.
 
-#### Inhabit a spot on the page
+#### :nut*and_bolt: Inhabit a page element *(advanced)\_
```

Comment:
Maybe it needs to be `:nut_and_bolt:` ?

Reply:
Also with our current recommended conventions, maybe `inhabit()` is "advanced" enough that it doesn't need to be in this section at all, but only in the abstract visualizer section below?  After all, I think it's now the case that neither of the two example visualizers use inhabit at all.

Reply:
I think this whole section is now more-or-less out of date; please respond either way to the suggestion of removing it altogether. Note we don't even mention depart() in this portion of the guide, either.

Reply:
Ah, I missed the thought about moving `inhabit()` entirely out of the example run-through. To me, it seems like `show()`, `stop()`, `depart()` are just as esoteric as `inhabit()`. Would you want to remove all of them, leaving "Error handling" as the only advanced section? I do agree that these sections all create friction. Having only the distinction between required and commonly used features would simplify reading too.

Reply:
A couple of reasonable options:
* Remove all of the "abstract visualizer" methods from this tutorial section, as you suggest.
* Have a single "advanced" section about the underlying abstract visualizer interface and the fact that you could extend these, referring to the "Abstract Visualizer" page or section for details, and maybe giving one example scenario that would lead you to use inhabit (like maybe having a CSS border on the containing div that changes color depending on aspects of the sequence? I dunno, that's pretty artificial, maybe you can come up with a better one).
* Or, if you think inhabit() is noticeably more likely to be of practical use to p5 visualizer writers, leave the other three in their nice short advanced section, and keep inhabit in its own advanced section that is up-to-date in its information, a succinct as practical, and definitely includes a plausible use case.

I would be fine with any of those presentations.

Reply:
Again, this is very nice now, thanks: the description of the possible-but-unlikely method inhabit() is nice and brief, and the details are elsewhere. Resolving.